src/items/qniteline.h

```diff
@@ -18,19 +20,23 @@ class QniteLine : public QniteXYArtist {
   void clearSelection() Q_DECL_OVERRIDE;
 
   bool drawSymbols() const { return m_drawSymbols; }
+  bool drawStepped() const { return m_drawStepped; }
   void setDrawSymbols(bool drawSymbols);
+  void setDrawStepped(bool drawStepped);
 
   QNanoQuickItemPainter *createItemPainter() const Q_DECL_OVERRIDE;
 
 signals:
   void drawSymbolsChanged();
+  void drawSteppedChanged();
 
 protected:
   virtual bool isSelected() const Q_DECL_OVERRIDE { return m_selected; }
 
 private:
   bool m_selected;
   bool m_drawSymbols;
+  bool m_drawStepped;
```

Comment:
Initialize this field in the constructor's initializer list, otherwise lines and splines could be rendered stepped when property is not explicitly set. 
Default should be set to `false`.